src/esp_littlefs.c

```diff
@@ -899,13 +899,21 @@ static int esp_littlefs_flags_conv(int m) {
 
 static void esp_littlefs_take_efs_lock(void) {
     if( _efs_lock == NULL ){
+#if portNUM_PROCESSORS > 1
```

Comment:
we're doing this because the 8266 doesn't have a `portENTER_CRITICAL` macro, correct? If so, I guess we should change this back to `#ifdef ESP8266` since this is an 8266-specific change, not dependent on the number of cores of the target device. Lets add a comment about it as well.

Sorry about the thrash!

Reply:
just searching the sdk makes it seem like `portENTER_CRITICAL` is available, however.

Reply:
Yes, just that ESP8266's `portENTER_CRITICAL` doesn't take any arguments -- probably just an outdated ESP-IDF convention, similar to the dir support. So, sure I guess we can just do `#ifdef ESP8266` -- so a future clean up (if the ESP8266 SDK ever get refreshed) would be easier. :D